src/SideNav.js

```diff
@@ -15,11 +15,12 @@ class SideNav extends Component {
   }
 
   render() {
-    const { className, children, fixed, ...props } = this.props;
+    const { className, children, ...props } = this.props;
     delete props.id;
     delete props.trigger;
     delete props.options;
-
+    delete props.fixed;
+    const fixed = this.props.fixed || !this.props.trigger;
     const classNames = cx('side-nav', { fixed }, className);
```

Comment:
```jsx
const classNames = cx('side-nav', className, {
  fixed: fixed || !trigger
});
```

Reply:
Didn't realize I could shorten this, this way. Thanks :)